compiler/compile/VirtualGuard.cpp

```diff
@@ -227,12 +227,12 @@ TR_VirtualGuard::createBreakpointGuardNode
 #ifdef J9_PROJECT_SPECIFIC
 /*
  * Shape of a breakpoint guard:
- * iflcmpeq/ificmpeq
+ * iflcmpne/ificmpne
  *    land/iand
  *       lloadi/iloadi <ConstantPool shadow>
  *          aconst J9Method
  *       isBreakpointedBit 
- *    isBreakpointedBit 
+ *    iconst 0 
  */
    bool is64Bit = TR::Compiler->target.is64Bit();
```

Comment:
Not a fan of having both this and the if..else.. now that the logic is getting blended - would be nice to simplify this down a bit.

Reply:
Thanks a lot for the suggestion. Modified the code to get rid of the ternary sign.